projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -5065,6 +5065,7 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         this.crudService.endEdit(true);
         this.selectionService.clearHeaderCBState();
         this.summaryService.clearSummaryCache();
+        this.pipeTrigger++;
```

Comment:
I feel like `pipeTrigger` in this case would be too much triggering all pipes, since it is already triggered once in the `endAddRow` in the `crud.servise.ts`. I see there is a `summaryPipeTrigger` which might suit better in this case.